ansible/roles/test/files/ptftests/fast-reboot.py

```diff
@@ -265,23 +460,63 @@ def runTest(self):
 
         self.log("Wait until ASIC stops")
         self.timeout(self.task_timeout, "DUT hasn't stopped in %d seconds" % self.task_timeout)
-        no_routing_start = self.check_stop()
+        no_routing_start, upper_replies = self.check_stop()
         self.cancel_timeout()
 
         self.log("ASIC was stopped, Waiting until it's up. Stop time: %s" % str(no_routing_start))
         self.timeout(self.task_timeout, "DUT hasn't started to work for %d seconds" % self.task_timeout)
-        no_routing_stop = self.check_start()
+        no_routing_stop, _ = self.check_start()
```

Comment:
I defined the exit criteria as working dataplane. I can check that all bgp session are up at the end of the test.